tools/fpgainfo/errors.c

```diff
@@ -400,8 +444,15 @@ static void print_errors_info(fpga_token token, fpga_properties props,
 			       error_value);
 
 			if (!strcmp(errinfos[i].name, "Errors")) {
-				size = PORT_ERROR_COUNT;
-				error_string = PORT_ERROR;
+				uint64_t revision = 0;
+				get_port_error_revision(token, &revision);
```

Comment:
Do we or should we bail here if we can't successfully determine the revision?

Reply:
My theory is that if the revision cannot be determined, then assume the original version.

I suppose another approach is that we don't decode at all if there is no revision.

Reply:
Either way (if we decode or not), I think we should print out an error/warning to stderr indicating that revision couldn't be determined.